server/helpers/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @description Get the app base url
+ *
+ * @param {object} httpObj HTTP request object
+ * @returns {string} the base url
+ * @methodget BaseUrl
+ */
+
+export const getBaseUrl = httpObj => {
+  return `${httpObj.protocol}://${httpObj.get("host")}/api/v1`;
+};
+
+/**
+ * @description Removes all white space character, if any
+ *
+ * @method cleanString
+ * @param {Strng} string The string to remove white space(s)
+ * @param {Boolean} removeAll: if false, multiple white space will reduce to single;
+ * @returns {Object} An object with trimed attributes
+ * @method cleanString
+ */
+export const cleanString = (string = "", removeAll) => {
```

Comment:
Benny left feedback about this `cleanString` function on Kingsley's PR. Please check it and implement it here as well